test/test_connection.py

```diff
@@ -53,6 +53,25 @@ def test_wrong_login():
         connection.login(user_pwd, user_pwd)
 
 
+def test_open_by_guest_negative():
+    connection = Connection(server_url)
+    connection.login(user_login, user_pwd)
+    name = connection.application('genestack/signin').invoke('whoami')
+    assert name == user_login
+
+
+def test_open_by_guest_positive():
```

Comment:
Why here we don't use SignIn app?

Reply:
Looks like we should remove guest word and replace it with something else.

Reply:
"unauthenticated"?

Reply:
We don't have dedicated wrapper for `SignIn`.